device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientOptions.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.device;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public final class ClientOptions
```

Comment:
Add a description to this class. Something about how it is for constructor time options that won't change for as long as this client exists